stackstorm-nexus3/config.schema.yaml

```diff
@@ -0,0 +1,31 @@
+---
+<<: &profile_options
```

Comment:
I totally approve of YAML tricks to reduce repeating code, but...

Just a warning: there might be a bug in StackStorm that won't process this correctly.

More information is here: https://github.com/StackStorm-Exchange/stackstorm-jira/pull/19#discussion_r241917265

But I do think this will be fixed in a future release of StackStorm (hopefully by 3.0).

Reply:
sure. Removed shorthand notation from schema